services/QuillLMS/client/app/bundles/Shared/components/shared/dataTable.tsx

```diff
@@ -255,14 +247,22 @@ export class DataTable extends React.Component<DataTableProps, DataTableState> {
 
   renderRowCheckbox(row) {
     const { showCheckboxes, checkRow, uncheckRow} = this.props
+    const handleClick = row.checked ? () => uncheckRow(row.id) : () => checkRow(row.id)
+    const ariaLabel = row.checked ? `Unselect ${row.name}` : `Select ${row.name}`
```

Comment:
I noticed that in some of the snapshot tests, this is resulting in "Unselect undefined" -- maybe we want to do this as a fallback:

```const ariaLabel = row.checked ? `Unselect ${row.name || 'row'}` : `Select ${row.name || 'row'}```